spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java

```diff
@@ -190,9 +222,24 @@ private Map setupPySparkEnv() throws IOException{
       }
     }
 
+    LOGGER.debug("PYTHONPATH: " + env.get("PYTHONPATH"));
     return env;
   }
 
+  // Run python shell
+  // Choose python in the order of
+  // PYSPARK_DRIVER_PYTHON > PYSPARK_PYTHON > zeppelin.pyspark.python
```

Comment:
I'm not feeling strongly about this, my take is for someone migrating environment from Spark only setup, they might feel the need to override Spark only setting from within Zeppelin (different virtualenv for example) such that both Spark and Spark in Zeppelin will work at the same time with one set of environment.

IMO having PYSPARK_DRIVER_PYTHON overriding zeppelin.pyspark.python is not very useful because if zeppelin.pyspark.python is not set PYSPARK_DRIVER_PYTHON is going to be used by spark-submit anyway.